File: JAVA/Pertemuan 01/src/id/ac/uph/pbo/pertemuan01/SegitigaSiku.java
package id.ac.uph.pbo.pertemuan01;

public class SegitigaSiku extends BangunDatar {
    private double alas, tinggi, sisiMiring;
    public SegitigaSiku(double alas, double tinggi) {
        super("Segitiga Siku-Siku");
        this.alas       = alas;
        this.tinggi     = tinggi;
        this.sisiMiring = Math.sqrt(alas * alas + tinggi * tinggi);
    }
    @Override public double hitungLuas()     { return 0.5 * alas * tinggi; }
    @Override public double hitungKeliling() { return alas + tinggi + sisiMiring; }
}

File: PYTHON/Pertemuan 01/SegitigaSiku.py
import sys
sys.stdout.reconfigure(encoding='utf-8')
import math
from BangunDatar import BangunDatar

class SegitigaSiku(BangunDatar):
    def __init__(self, alas: float, tinggi: float):
        super().__init__("Segitiga Siku-Siku")
        self.alas = alas
        self.tinggi = tinggi
        self.sisi_miring = math.sqrt(alas * alas + tinggi * tinggi)

    def hitung_luas(self) -> float:
        return 0.5 * self.alas * self.tinggi

    def hitung_keliling(self) -> float:
        return self.alas + self.tinggi + self.sisi_miring

